src/types/Gene.jl

```diff
@@ -4,17 +4,10 @@ $(TYPEDEF)
 # Fields
 $(TYPEDFIELDS)
 """
-mutable struct Gene
+Base.@kwdef mutable struct Gene
```

Comment:
Put this in a lot of places, slightly unrelated to the PR, but nice to use!

Reply:
yeah certainly helps but we should have a PR that does it properly and globally. E.g. here you can move the defaults from the constructor to the struct definition.

Reply:
( oh okay you did already, good. :D )